tests/v2/validation/registries/registry_test.go

```diff
@@ -91,27 +116,32 @@ func (rt *RegistryTestSuite) SetupSuite() {
 	providers := clustersConfig.Providers
 	rt.advancedOptions = clustersConfig.AdvancedOptions
 
-	globalRegistryFqdn := ""
 	rt.rancherUsesRegistry = false
-	globalRegistryFqdn, err = corral.GetCorralEnvVar(corralRancherName, "registry_fqdn")
+	listOfCorrals, err := corral.ListCorral()
 	require.NoError(rt.T(), err)
-	if globalRegistryFqdn != "<nil>" {
-		rt.rancherUsesRegistry = true
-		logrus.Infof("Rancher Global Registry FQDN %s", globalRegistryFqdn)
+	_, corralExist := listOfCorrals[corralRancherName]
+	if corralExist {
+		globalRegistryFqdn, err = corral.GetCorralEnvVar(corralRancherName, "registry_fqdn")
+		require.NoError(rt.T(), err)
+		if globalRegistryFqdn != "<nil>" {
+			rt.rancherUsesRegistry = true
+			logrus.Infof("Rancher Global Registry FQDN %s", globalRegistryFqdn)
```

Comment:
This makes the PR dependent of my external registries? or whichever goes first resolve the conflicts 😄

Reply:
Yup but your PR will take precedent and be merged first.

Reply:
your PR was merged